build/yamls/chart-values/antrea-ipsec.yml

```diff
@@ -1,3 +1,5 @@
 trafficEncryptionMode: "ipsec"
 # change the tunnel type to GRE which works better with IPsec encryption than other types.
 tunnelType: "gre"
+multicast:
+  enabled: false
```

Comment:
not really needed
we have no plan of enabling `multicast.enabled` by default

Reply:
Got it, removed.